react-native-sdk/index.tsx

```diff
@@ -37,10 +36,19 @@ interface IAppProps {
     style?: Object;
 }
 
+/**
+ * Converts the meetingOptions domain and roomName to a URL that can be passed to the App component.
+ * @param {*} domain domain address from props.
+ * @param {*} roomName room name from props.
+ */
+const convertPropsToURL = (domain, roomName) => {
```

Comment:
Do we really need this function? Can't we in-line it?

Reply:
Yeah, we could.